docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -748,6 +750,41 @@ In this example, the table is partitioned by the month of ``order_date``, a hash
         country VARCHAR)
     WITH (partitioning = ARRAY['month(order_date)', 'bucket(account_number, 10)', 'country'])
 
+Sorted tables
+------------------
+
+In addition to partitioning, Iceberg tables may have a specified list of sort columns. When writes
+are made to the table, each file will individually be sorted by these columns. Sorting can be
+particularly helpful when selective reads are frequently made against a column with a high cardinality.
+
+Sort orders are provided using the ``sorted_by`` table property. For example::
+
+    CREATE TABLE iceberg.testdb.customer_orders (
+        order_id BIGINT,
+        order_date DATE,
+        account_number BIGINT,
+        customer VARCHAR,
+        country VARCHAR)
+    WITH (partitioning = ARRAY['country'], sorted_by = ARRAY['order_date'])
+
+Sorting can be combined with partitioning on the same column, for example if a timestamp
+column is often queried selectively but the data is sparse enough that a higher granularity
+partition transform produces a better data layout::
```

Comment:
Is it worth clarifying that we don't mean "identity partitioning" (so not "any partitioning") here?

"data is sparse enough" -- i don't understand the condition
if data is very sparse, you won't have many rows per partition, and sorting won't help
would "is dense enough" be considered appropriate?

Reply:
in either case "enough" is not really a concrete value .. more like an estimate .. we should try to be more definitive.

Reply:
> if data is very sparse, you won't have many rows per partition, and sorting won't help

That's the condition I was trying to describe. A timestamp column is sparse enough that you want to lump en entire year worth of data into a partition but predicates are still using day or hour precision.

Reply:
Do we even need to explain when this is beneficial .. maybe not .. maybe we can just say that you can do it. And bring the example ..